egs/wham/ConvTasNet/local/prepare_data.sh

```diff
@@ -20,11 +21,9 @@ mv $out_dir/wham_scripts.tar.gz $out_dir/wham_scripts
 
 wait
 
-unzip $out_dir/wham_noise.zip $out_dir >> logs/unzip_wham.log
-
 echo "Run python scripts to create the WHAM mixtures"
 # Requires : Numpy, Scipy, Pandas, and Pysoundfile
-cd $out_dir/wham_scripts
+cd $out_dir/wham_scripts/wham_scripts
```

Comment:
My bad, I added an additional folder level in vain for the untar. Let's change the lines 18-20 instead, it makes more sense.

Reply:
ok